src/index.ts

```diff
@@ -1,17 +1,41 @@
+const fs = require('fs');
+
 const core = require('@actions/core');
 const exec = require('@actions/exec');
+// const childProcess = require('child_process');
+
+const request = require('request');
+
+// import buildExec from './buildExec';
+
+// const {failCi} = buildExec();
 
-import buildExec from './buildExec';
+try {
+  const filename = __dirname + '/uploader';
+  request.get('https://uploader.codecov.io/latest/codecov-linux', (err, res, body) => {
```

Comment:
```suggestion
  request.get('https://raw.githubusercontent.com/codecov/codecov-action/master/codecov', (err, res, body) => {
```

The updated URL points to the Bash script in *this* repository.

Reply:
👍 I should have thought of that, but the issue seems toa actually be with the new uploader, I'm going to noodle some more.